src/ProgramBlockEditor.scss

```diff
@@ -134,6 +134,7 @@ button.btn.ProgramBlockEditor__program-deleteAll-button--pressed.ProgramBlockEdi
     display: flex;
     justify-content: center;
     padding: 0 0.5rem;
+    padding-top: 2.9rem;
```

Comment:
With this padding top, the icon may look aligned with program sequence, but I think we should apply same style that's applied in the program sequence. The program sequence uses padding-bottom: 1rem to position itself, so what we can do here is to make the icon start from the bottom of the character column container, and then use bottom-padding: 1rem as well. Let's update .Program__character-column style to be:

```
.ProgramBlockEditor__character-column {
    align-items: center;
    background-color: #30444E;
    border-right: 1px solid #19282F;
    border-radius: 3px 0 0 3px;
    box-shadow: 2px 0px 1px #8F8F8F;
    display: flex;
    flex-direction: column;
    justify-content: flex-end;
    padding-bottom: 1rem;
}
```

Reply:
This did not work for me.  Were there other changes required?

![Screenshot 2021-11-16 at 11 46 34](https://user-images.githubusercontent.com/5765222/141971500-582a1e4e-fc3b-4023-982c-8e46e16502db.png)